net-vpn/i2p/i2p-0.9.33.ebuild

```diff
@@ -0,0 +1,164 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils java-pkg-2 java-ant-2 systemd user
+
+DESCRIPTION="A privacy-centric, anonymous network."
```

Comment:
No full stop, this is not a full sentence.

Reply:
Done.